Sources/ContainerPersistence/DefaultsStore.swift

```diff
@@ -26,6 +26,8 @@ public enum DefaultsStore {
         case buildRosetta = "build.rosetta"
         case defaultDNSDomain = "dns.domain"
         case defaultBuilderImage = "image.builder"
+        case defaultBuilderStorage = "builder.storage"
```

Comment:
Why is it storage in one and capacity in another? How about `builder.storage` and `container.storage`?